types/sandbox/prerequest.d.ts

```diff
@@ -14,8 +14,18 @@ declare interface PostmanLegacy {
     setNextRequest(requestName: string): void;
 }
 
+/**
+ * @param execution - -
+ * @param onRequest - -
+ * @param onAssertion - -
+ * @param cookieStore - -
+ * @param [options] - -
+ * @param [options.disabledAPIs] - -
+ */
```

Comment:
Let's get rid of the extra diff.

Reply:
That would mean removing jsdocs from places as this type script is auto generated. I am not sure if we should be doing that, best we can do is add jsdoc description to public apis.